lib/internal/modules/esm/load.js

```diff
@@ -29,13 +29,12 @@ const {
  * @param {ESModuleContext} context used to decorate error messages
  * @returns {Promise<{ responseURL: string, source: string | BufferView }>}
```

Comment:
No longer returns a `Promise`:
```suggestion
 * @returns {{ responseURL: string, source: string | BufferView }}
```